docs/admin/config.rst

```diff
@@ -648,11 +648,13 @@ Default setting for translation propagation, defaults to ``True``.
 
 .. setting:: DEFAULT_PULL_MESSAGE
 
+.. _config-pull-message:
+
 DEFAULT_PULL_MESSAGE
 --------------------
 
-Title for new pull requests,
-defaulting to ``'Update from Weblate'``.
+Configures the default title and message for pull requests.
+Title defaults to ``Translations update from {{ site_title }}``.
```

Comment:
Maybe it's not a good idea to include the default here - it will easily get out of sync as happened now...

Reply:
Yeah, I was wondering about that myself (especially as it was already an old state), if you want I will exclude it completely?
IMO it doesn't make to much sense anyway as pasting the entire message is way too much and the default title doesn't contain that much valuable information to the end user. If they try to configure this value they've probably already seen the default in action anyway?

Reply:
I'd completely ommit it.

Reply:
👍 I've removed the default value and force pushed.